src/JsonSchema/Uri/UriResolver.php

```diff
@@ -80,43 +82,45 @@ public function resolve($uri, $baseUri = null)
 
         $components = $this->parse($uri);
         $path = $components['path'];
-        
-        if (! empty($components['scheme'])) {
+
+        if (!empty($components['scheme'])) {
             return $uri;
         }
         $baseComponents = $this->parse($baseUri);
         $basePath = $baseComponents['path'];
-        
+
         $baseComponents['path'] = self::combineRelativePathWithBasePath($path, $basePath);
         if (isset($components['fragment'])) {
             $baseComponents['fragment'] = $components['fragment'];
         }
 
         return $this->generate($baseComponents);
     }
-    
+
     /**
      * Tries to glue a relative path onto an absolute one
      *
      * @param string $relativePath
      * @param string $basePath
-     * @return string Merged path
+     *
      * @throws UriResolverException
+     *
+     * @return string Merged path
      */
     public static function combineRelativePathWithBasePath($relativePath, $basePath)
     {
         $relativePath = self::normalizePath($relativePath);
         if ($relativePath == '') {
             return $basePath;
         }
-        if ($relativePath{0} == '/') {
+        if ($relativePath[0] == '/') {
```

Comment:
As far as I know, it is a string and array access is merely used as a quick way to check the first character.